appinfo/app.php

```diff
@@ -19,5 +19,5 @@
 $eventDispatcher->addListener('OCA\Files_Sharing::loadAdditionalScripts', function () {
 	OC_Util::addVendorScript('core', 'marked/marked.min');
 	OCP\Util::addScript('files_texteditor', 'public-share');
-	OCP\Util::addStyle('files_texteditor', '../build/public-share');
+	OCP\Util::addStyle('files_texteditor', 'public-share');
```

Comment:
Looks like a wrong c&p because only JS is placed in there. SCSS is processed by NC and thus already at the correct location.